connexion/utils.py

```diff
@@ -245,3 +245,17 @@ def create_empty_dict_from_list(_list, _dict, _end_value):
         return {current_key: create_empty_dict_from_list(_list, _dict, _end_value)}
     else:
         return {current_key: _end_value}
+
+
+def _immutable_pop(_dict, key):
```

Comment:
This is only used once - I don't think it should be in utils.

Reply:
also FYI - the leading underscore is a convention that means that the function is private (not meant to be used outside of a class / module), so moving it to utils and keeping the underscore is confusing.